.gitignore

```diff
@@ -5,3 +5,15 @@
 _build/
 _sbuild/
 *.o
+*.a
+
+# temp files
+*.swp
+*.swo
```

Comment:
These belong in your system's global gitignore, not in the project's. Otherwise we'll have 100 lines covering everyone's favourite editor.

Reply:
Gah. I missed this. Thanks, I'll get rid.